### PR TITLE
feat(api): add badges endpoint

### PR DESCRIPTION
**Pull request by sileht**
Fixes MRGFY-688

Change-Id: Ief6665d27f161f701084ade646684d6dc4c87b40

### CODE REVIEW

File: mergify_engine/web/api/queues.py [hunk 1]
```diff
@@ -36,6 +36,13 @@
 
 LOG = daiquiri.getLogger(__name__)
 
+router = fastapi.APIRouter(
+    tags=["queues"],
+    dependencies=[
+        fastapi.Depends(security.require_authentication),
+    ],
+)
+
```
Comment (jd) [inline mergify_engine/web/api/queues.py:45]: unrelated?

**Review by jd — APPROVED** [review]

File: mergify_engine/web/api/queues.py (hunk 1 above)
Comment (sileht) [reply mergify_engine/web/api/queues.py:45]: It's related, I relax the default security dependencies of /v1 endpoint, I move them to queues/applications routers instead. So /v1/badges don't need authentication and this is reflexed in OpenAPI generated file.
